corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -45,7 +45,49 @@ cloudCare.Case = Backbone.Model.extend({
     
     status: function () {
         return this.get("closed") ? "closed" : "open";
-    }
+    },
+
+    caseProperties: function(language) {
+        // Returns case-details as key-value pairs
+        // there should be a better way to access language var, than passing here
+        var raw_columns = this.get("module") ?
+                          this.get("module").get("case_details").long.columns :
+                          this.get("appConfig").module.get("case_details").long.columns ; // If Parent-child selection is on
+        return _.map(raw_columns, function(col){
+            return {
+                key: localize(col.header, language),
+                value: this.getProperty(col.field) ? this.getProperty(col.field) : cloudCare.EMPTY
+            };
+        }, this);
+    },
+
+    caseDetailsLabel: function(language) {
+        var caseLabel = (this.get("module") ?
+                         this.get("module").get("case_label")[language] :
+                         this.get("appConfig").module.get("case_label")[language]) + 
+                        ": ";
+        if (caseLabel === "Cases: "){
+            caseLabel = "";
+        }
```

Comment:
Looks like this was copied over from functionality that I previously implemented.
The idea behind blanking "Cases: " is just to make the breadcrumb a little more readable. If the label for your case type is "Patients," then the breadcrumb would look something like this: `"Selected Case: Patients: John"`, but if you leave the label with the default "Cases", then it would look like this: `"Selected Case: Cases: John"`. Since the bread crumb is already prefixed with "Selected Case:", the extra "Cases: " in the middle there seems redundant. There might be a way to make it more translation friendly though.